app/src/main/java/com/github/wnder/GuessPreviewActivity.java

```diff
@@ -72,14 +72,10 @@ protected void onCreate(Bundle savedInstanceState) {
         imageDisplayed = findViewById(R.id.imagePreview);
 
         //Setup buttons
-        findViewById(R.id.SaveToGallery).setOnClickListener(id -> saveToGallery());
+        findViewById(R.id.guessButton).setOnClickListener(id -> openGuessActivity());
 
-
-
-        //Setup swipe and click action
-        imageDisplayed.setOnClickListener(view -> {
-            openGuessActivity();
-        });
+        //Setup swipe
+        imageDisplayed.setOnClickListener(id -> {});
```

Comment:
Is this still needed? I guess it might be to avoid problem with swipe, just want to make sure

Reply:
Yes, if this line is removed, the espresso function to swipe in the test does nothing. It's a little bit strange, but it only impacts the test so I left it here...